src/css/header.css

```diff
@@ -1,4 +1,291 @@
+@import url('./font.css');
```

Comment:
dont need this, it's alredy imported
![image](https://github.com/user-attachments/assets/f0d793da-ffb0-4c91-a45d-208e6130e8ab)